test/images/image-util.sh

```diff
@@ -99,6 +99,10 @@ push() {
     TAG=$(<${IMAGE}/VERSION)
     docker push ${REGISTRY}/${IMAGE}-${arch}:${TAG}
   done
+
+  # Make archs list into OS/architecture pair. Eg: 'amd64 ppc64le' to 'linux/amd64,linux/ppc64le'
+  archs=$(echo $archs | sed -e 's/[^ ]* */linux\/&/g' -e 's/ /,/g')
```

Comment:
neat sed trick: to avoid having to escape the slash, you can actually use (nearly) any character as a separator, not just `/`.

i.e. this could be written
`sed -e 's|[^ ]* *|linux/&|g' -e 's/ /,/g'`

Reply:
(in case it's not clear, I'm not blocking this PR on this change, just wanted to give you an FYI in case you didn't know.)

Reply:
@ixdy : Thanks for the review and the tip!